CommunicationLayer/CommDeviceControl/UdpMessageForwarder.cpp

```diff
@@ -26,27 +26,39 @@
 #include <QByteArray>
 #include <QDebug>
 #include <QHostAddress>
+#include <QTextCodec>
+
 
 #include "InfrastructureLayer/Settings/I_Settings.h"
 #include "UdpMessageForwarder.h"
 
+namespace
+{
+    const QString EXCHANGE_NAME = QString("amq.direct");
+}
+
 UdpMessageForwarder::UdpMessageForwarder(I_Settings& settings)
-: ipAddress_(settings.ipAddress())
-, port_(settings.udpPort())
 {
-    socket_.bind(ipAddress_, port_);
+  QString queueName = settings.queueName();
+
+  routingKey = settings.routingKey();
+
+  channel = Channel::Create(settings.ipAddress().toStdString(), (int)settings.udpPort());
+
+  channel->DeclareQueue(queueName.toStdString(), false, true, false, false);
```

Comment:
I know one could just look up the documentatin, but perhaps a comment here explaining the line of true/false would save some time haha.